### PR TITLE
Cleaned up warnings and provided some error checking in build.php

### PR DESCRIPTION
**Pull request by deanhouseholder**
Added a try/catch block to minimize PHP fatal error output while retaining the message.

### CODE REVIEW

File: build.php [hunk 1]
```diff
@@ -28,12 +28,22 @@
...
+if (\file_exists($outpath)) {
+    \unlink($outpath);
+}
+
+try {
+    $phar = new Phar($outpath);
+} catch (Exception $e) {
+    echo 'PHP Fatal error: '.$e->getMessage();
+    die;
+}
```
Comment (jnvsor) [inline build.php:45]: This try catch is redundant, since the exception will result in a fatal error anyway (And a real one, with the proper exit code, print to stderr, etc)

File: build.php [hunk 2]
```diff
@@ -55,3 +65,5 @@
 $phar = new Timestamps($outpath);
 $phar->updateTimestamps();
 $phar->save($outpath, Phar::SHA512);
+
+echo 'Your kint.phar has been built successfully and is located in the "build" directory.';
```
Comment (jnvsor) [inline build.php:69]: Since there's no output there's a future possibility for us to pipe the script to a file instead of writing directly, I think we should just leave stdout empty for now

**Review by jnvsor — CHANGES_REQUESTED** [review]
> I'm going to cancel and submit a new one.

FYI: You can force push to your branch to automatically update the PR, no need to make a new one

**Comment by raveren** [comment]
Is this maybe why our whole build is broken for weeks :DD @jnvsor I don't think we should keep this open, @deanhouseholder thanks for the effort, and sorry we couldn't make use of the code, but people like you is who inspire us to build things like this.

**Comment by jnvsor** [comment]
@raveren No, the build isn't broken. @deanhouseholder I'd like to reopen this, but if you don't feel like finishing it up I'll leave it closed.

**Comment by deanhouseholder** [comment]
I can't understand why you prefer ugly errors to clean ones. Haha

**Comment by jnvsor** [comment]
This gets a bit unixy as far as technicals go, but programs with text input/output generally have 3 streams: stdin (input) stdout (output) and stderr (error)

Why do you have different streams for output and error? Because it's common to connect programs up to eachother. For instance you could pipe a php script's output to another one like so:

```bash
php script.php | php script2.php
```

If you do it like this, only stdout (The output) is piped to script2, and stderr ends up in the terminal. Apache modphp typically sends both stdout and stderr to the browser, so it doesn't matter there.

The second reason is that programs return a number when they end indicating success or failure. A 0 is success and anything else indicates a variety of failures.

```bash
php script.php && php script2.php
```

In this example script2 will only run if script1 succeeded, which it knows by the exit code. A proper fatal error returns a non-0 exit code, which is also used by CI to tell if things succeeded.

In other words by manually exiting the script *exceptions in the build won't be caught by CI* which is a very bad thing.

Long story short: Smarter people than us thought up these systems and they work very well. No touchy :)
